.github/workflows/amplify_canaries.yaml

```diff
@@ -1,6 +1,7 @@
 name: Amplify Canaries
 on:
   schedule:
+    # TODO(kylechen): DO NOT MERGE until clarified runtime frequency
```

Comment:
We need to discuss how often the canary should run. 

The UI team canaries run every 15 minutes 24/7. 

### UI team canary rough cost estimate: 
1. Each canary action takes ~5 minutes: https://github.com/aws-amplify/amplify-ui/actions/runs/5292184239
2. 24 hours / 15 minutes => 96 runs / day 
3. Unix cost multiplier = x1 
4. Total cost / day = 5 * 96 * 1 = 480 

### Amplify Flutter canaries cost estimate: 
1. Each canary action takes ~55 minutes: https://github.com/aws-amplify/amplify-flutter/actions/runs/5290286906
2. 24 hours / 15 minutes => 96 runs / day 
3. MacOS cost multiplier = x10 
4. Total cost / day = 55 * 96 * 10 = 52800

If we were to match UI team behavior our costs would be ~ x110 theirs. 

If we were to run actions at a frequency that matches their "480" cost it would be: 

48/55 runs a day (vs their 96 runs a day)

Reply:
Daily seems appropriate for our canaries. Our repo doesn't change that frequently and half the tests run on published versions.

We may actually want to split out the tests that use published versions to run more infrequently. Getting constant reminders that a published library is broken doesn't seem super helpful, especially if this is to somehow create SIM tickets.